roles/setup_repo/tasks/PG_RedHat_setuprepos.yml

```diff
@@ -142,6 +142,14 @@
     state: present
   become: true
 
+# Exclusive to pgd setup authentication
+# Package is used for ident authentication
+- name: Install oidentd package if not available
+  ansible.builtin.package:
+    name: oidentd
+    state: present
+  become: true
```

Comment:
Add `when: install_pgd`

Reply:
This package on RHEL systems is also only in the EPEL repo, so the conditional should actually be
```
when:
  - install_pgd
  - enable_epel_repo
```